lib/app/components/organisms/talk_member_card.dart

```diff
@@ -14,57 +16,68 @@ class TalkMemberCard extends StatelessWidget {
         ),
-      ),
-      child: Padding(
-        padding: const EdgeInsets.all(15),
-        child: Row(
-          children: [
-            CircleAvatar(
-              //TODO:firebase接続後に動作確認
-              //profileImageURLがあればその画像を表示、なければグレー背景でpersonアイコンを表示
-              foregroundImage: profileImageURL != null
-                  ? NetworkImage(profileImageURL!)
-                  : null,
-              backgroundColor: Colors.grey,
-              child: const Icon(
-                Icons.person,
-                color: Colors.white,
+        child: Padding(
+          padding: const EdgeInsets.all(15),
+          child: Row(
+            children: [
+              CircleAvatar(
+                //TODO:firebase接続後に動作確認
+                //profileImageURLがあればその画像を表示、なければグレー背景でpersonアイコンを表示
+                foregroundImage: profileImageURL != null
+                    ? NetworkImage(profileImageURL!)
+                    : null,
+                backgroundColor: Colors.grey,
+                child: const Icon(
+                  Icons.person,
+                  color: Colors.white,
+                ),
+              ),
+              const SizedBox(
+                width: 20,
               ),
-            ),
-            const SizedBox(
-              width: 20,
-            ),
-            Flexible(
-              child: Column(
-                mainAxisSize: MainAxisSize.min,
-                mainAxisAlignment: MainAxisAlignment.center,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  Text(
-                    roomName,
-                    maxLines: 1,
-                    overflow: TextOverflow.ellipsis,
-                    style: const TextStyle(fontSize: 20),
-                  ),
-                  Text(
-                    mostRecentMessage,
-                    maxLines: 2,
-                    overflow: TextOverflow.ellipsis,
-                  ),
-                ],
+              Flexible(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Text(
+                      roomName,
+                      maxLines: 1,
+                      overflow: TextOverflow.ellipsis,
+                      style: const TextStyle(fontSize: 20),
+                    ),
+                    Text(
+                      mostRecentMessage,
+                      maxLines: 2,
+                      overflow: TextOverflow.ellipsis,
+                    ),
+                  ],
+                ),
               ),
-            ),
-          ],
+            ],
+          ),
         ),
       ),
     );
   }
+
+  //fixme: firebaseからプロフィールを受け取れるようになったら引数見直し
+  void _moveToTalkRoom(String roomName, String? profileImageURL) {
+    Get.toNamed(
+      '/talk-room',
```

Comment:
```suggestion
      Routes.talkroom,
```
これでチャットページのパス名を呼び出せると思うので修正お願いします。
あと、’Get.toNamed’は分離しないで呼び出し元に書いちゃっていいですよ。